docs/ref/contrib/admin/index.txt

```diff
@@ -1326,15 +1326,18 @@ templates used by the :class:`ModelAdmin` views:
 
 .. warning::
 
-    :meth:`ModelAdmin.save_model` and :meth:`ModelAdmin.delete_model` must
-    save/delete the object, they are not for veto purposes, rather they allow
-    you to perform extra operations.
+    When implementing :meth:`ModelAdmin.save_model` and
```

Comment:
I chose "overriding" specifically to call out that this is _not_ a hook that gets called during the save process but rather overrides the parent class's implementation. This was my confusion in the first place.

I'd prefer the use of the word "overriding".